streams/writable-streams/aborting.js

```diff
@@ -1092,21 +1091,20 @@ promise_test(t => {
   const abortPromise = ws.abort('done');
   controller.error(error1);
   resolveStart();
-  return promise_rejects(t, error1, abortPromise, 'abort() should reject if start() errors the controller')
-      .then(() =>
-      assert_array_equals(ws.events, [], 'abort() should be not be called if start() errors the controller'));
-}, 'abort() promise should reject if start() errors the controller');
+  return abortPromise.then(() =>
+      assert_array_equals(ws.events, ['abort', 'done'],
+                          'abort() should still be called if start() errors the controller'));
+}, 'if start attempts to error the controller after abort() has been called, then it should lose');
 
-promise_test(t => {
+promise_test(() => {
   const ws = recordingWritableStream({
     start() {
       return Promise.reject(error1);
     }
   });
-  return promise_rejects(t, error1, ws.abort('done'), 'abort() should reject if start() rejects')
-      .then(() =>
-      assert_array_equals(ws.events, [], 'abort() should be not be called if start() rejects'));
-}, 'stream abort() promise should reject if sink start() rejects');
+  return ws.abort('done').then(() =>
+      assert_array_equals(ws.events, ['abort', 'done'], 'abort() should still be called if start() rejects'));
```

Comment:
This is unfortunate, but I don't feel that it is serious enough to require a note in the standard. I have opened https://github.com/whatwg/streams/issues/735 to track it.